TrackParamFilter/sections/specific.txt

```diff
@@ -8,6 +8,9 @@
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/151756
 ||pravda.com.ua^$removeparam=pageviewCount
+https://github.com/AdguardTeam/AdguardFilters/pull/152161
+||belta.co.jp^$removeparam=cid
+||belta.co.jp^$removeparam=urrad
```

Comment:
`urrad` is also here
![image](https://github.com/AdguardTeam/AdguardFilters/assets/8361299/174e6ed6-f0bb-42a2-b817-6ebeb3af14db)

Reply:
I will add that domain.